lte/gateway/python/setup.py

```diff
@@ -125,7 +125,7 @@
         'freezegun>=0.3.15',
         'pycryptodome>=3.9.9',
         'pyroute2==0.5.14',
-        'aiohttp==3.6.2',
+        'aiohttp==3.8.0',
```

Comment:
```suggestion
        'aiohttp',
```
Unfortunately due to our VMs mismatch on python versions (magma VM on 3.8 and magma_test VM on 3.5) the `aiohttp` 3.8.0 package is not available on the test VM (maximum version that comes up is 3.6.3), so we can leave this package without version so on the magma VM the latest version is installed.

Note: This will be resolved eventually when we migrate to a containerized versions for our dev and testing environments, cc @arunuke

Reply:
I made changes...